.github/workflows/check_wheel_availability.yaml

```diff
@@ -19,20 +19,32 @@ jobs:
   check_dependency_artifacts:
     name: Check Dependency Artifacts
     runs-on: ${{ matrix.os }}
+    container: ${{ matrix.os == 'arm64' && format('docker://python:{0}', matrix.python-version) || '' }}
     strategy:
       fail-fast: false
       matrix:
         python-version: ['3.7', '3.8', '3.9']
-        os: [macOS-latest, ubuntu-latest]
+        os: [macos-latest, m1, ubuntu-latest, arm64]
+        exclude:
+          - os: macos-latest
+            python-version: 3.7
+          - os: m1
+            python-version: 3.7
```

Comment:
Shouldn't this be 3.8?

Reply:
I should read "exclude" when it's exclude and not include...

Reply:
This is an exclude of 3.7 from the matrix.  In the 3.10 PR this should remain but also 3.8 should be added to the exclusions. 
 Admittedly exclude vs. include is a bit of an option now with just two platforms but when we add Windows using the exclude option will make more sense.  With a bit more information in the matrix this could be a single exclude rather  than two.